### ISSUE TITLE
Y axis max value

### ISSUE THREAD

**Issue by seabbs** [hamishgibbs/rt_vis#59]
Should be min(max(observed data in window, top of ribbon in window) + a small amount, 5 * max(observed data in window)) I think

**Comment by seabbs** [comment]
Should hopefully avoid this sort of issue where blank space is most of the plot. 

![Screenshot 2020-10-05 at 16 42 30](https://user-images.githubusercontent.com/20317649/95101175-c4b23900-0729-11eb-91ba-d11fd2e8181e.png)

**Comment by hamishgibbs** [comment]
I believe the 5 * max_obs_cases is currently defined by the max for that country (or region) overall but should be defined by the max for the country (or region) in the active time window.

**Comment by seabbs** [comment]
I don't think it is as the y axis updates as you filter. I think it just assumes that 5* observed cases is near the ribbon but sometimes we manage to not have such mad uncertainty 😆

**Comment by hamishgibbs** [comment]
Ah that's true. Ok I will look into what is going on. Thanks!
